core/PRP/ConditionalObject/plKeyPressConditionalObject.h

```diff
@@ -36,6 +36,10 @@ class PLASMA_DLL plKeyPressConditionalObject : public virtual plConditionalObjec
 protected:
     virtual void IPrcWrite(pfPrcHelper* prc);
     virtual void IPrcParse(const pfPrcTag* tag, plResManager* mgr);
+
+public:
+    plKeyDef getKeyEvent() const { return fKeyEvent; }
```

Comment:
That works for the ones with a direct ASCII value (e.g. 'A' or '0'), but there are also some special values (e.g. KEY_F1) which have no obvious value.